x/valaddr/client/cli/tx.go

```diff
@@ -0,0 +1,67 @@
+package cli
+
+import (
+	"github.com/celestiaorg/celestia-app/v6/x/valaddr/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd returns the transaction commands for the valaddr module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Valaddr transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		CmdSetFibreProviderInfo(),
+	)
+
+	return cmd
+}
+
+// CmdSetFibreProviderInfo broadcasts a MsgSetFibreProviderInfo transaction
+func CmdSetFibreProviderInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-provider-info [host]",
```

Comment:
```suggestion
		Use:   "set-host [host]",
```
This is what I've used in the spec to keep terminology consistent